### PR TITLE
Support geospatial file upload of land boundaries

### PR DESCRIPTION
**Pull request by pwadmore-ea**
https://eaflood.atlassian.net/browse/BNG-914

### CODE REVIEW

**Review by teddmason — APPROVED** [review]
I left a few comments but nothing critical at all

File: packages/application-to-register-webapp/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        }
+      }
+    ]
+  },
+  plugins: [
+    new HtmlPlugin({
+      inject: false,
+      filename: '../views/land/confirm-land-boundary.html',
+      template: 'src/views/webpack/land/confirm-land-boundary.html',
+      chunks: ['confirmLandBoundary']
```
[the remark this answers is not in the document]
Comment (pwadmore-ea) [reply packages/application-to-register-webapp/webpack.config.js:39]: Agreed

File: packages/document-service/src/__tests__/service.spec.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Readable } from 'stream'
+import { upload } from '../service.js'
+
+describe('The document service', () => {
+  it('should delgate the upload of a document to a cofigured functon', async () => {
```
[the remark this answers is not in the document]
Comment (pwadmore-ea) [reply packages/document-service/src/__tests__/service.spec.js:5]: Agreed

File: packages/geoprocessing-service/src/service.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    // The published solution does not appear to be effective for unit tests
+    // associated with this module.
+    //
+    // Load the module that links to GDAL dynamically so that unit tests
+    // mocking this module do not load attempt to load the native module that
+    // links to GDAL.
+    //
+    // This allows the unit test file associated with this file to be the only
+    // unit test file that loads the native module linking to GDAL.
+    const gdal = (await import('gdal-async')).default
```
[the remark this answers is not in the document]
Comment (pwadmore-ea) [reply packages/geoprocessing-service/src/service.js:18]: I suggest performance implications of the dynamic import are something that needs monitoring.

If performance is really affected, another way of approaching the tests will need to be found.

**Review by teddmason — APPROVED** [review]

**Pull request merged**